src/shared/login-modal.jsx

```diff
@@ -37,6 +37,11 @@ export default class LoginModal extends React.Component {
             &nbsp; applies to your content. Your notebooks are associated with
             your account.
           </p>
+          <p>
+            Deleting a notebook permanently deletes its contents. If you wish to
```

Comment:
Do we need to add some nuance here about other users forking copies of a notebook? What about revisions?

Reply:
@wlach

Reply:
I thought about that, but then thought it was self-evident.  But maybe not.  "Deleting a notebook permanently deletes any copies and revisions in your account."?